spot_driver/spot_driver/ros_helpers.py

```diff
@@ -127,20 +128,21 @@ def createDefaulCameraInfo():
 
     return camera_info_msg
 
-def getImageMsg(data, spot_wrapper):
-    """Takes the imag and  camera data and populates the necessary ROS messages
+
+def bosdyn_data_to_image_and_camera_info_msgs(data: image_pb2.ImageResponse, spot_wrapper: SpotWrapper):
+    """Takes the image and camera data and populates the necessary ROS messages
     Args:
+        spot_wrapper: SpotWrapper
         data: Image proto
-        spot_wrapper: A SpotWrapper object
     Returns:
         (tuple):
             * Image: message of the image captured
             * CameraInfo: message to define the state and config of the camera that took the image
     """
     image_msg = Image()
-    local_time = spot_wrapper.robotToLocalTime(data.shot.acquisition_time)
+    local_time = spot_wrapper.robot_to_local_time(data.shot.acquisition_time)
```

Comment:
While stylistically correct, this seems like maybe an unnecessary change for this PR?  I'm worried we'll miss a call to it somewhere.

Reply:
I got all of them in the repository and the code still runs, but if you'd like me to undo it I will.

Reply:
I'm worried about external users like MASKOR.  @METEORITENMAX wdyt?

Reply:
I went ahead an undid these changes.